distribution/ddf-common/src/main/resources/etc/org.apache.karaf.command.acl.shell.cfg

```diff
@@ -0,0 +1,17 @@
+#
+# This configuration file defines the ACLs for commands in the shell subshell
+#
+
+* = NO_ACCESS
+complete = admin
+echo = admin
+format = admin
+grep = admin
+if = admin
+keymap = admin
+less = admin
```

Comment:
Do we actually still need less? Does this allow you to read files on the file system?

Reply:
No. It's used for paging output from other karaf commands.

Reply:
Ugh...but might provide access to system files. It might be a problematic shell command. Let me double check.